core/storage/suggestion/gae_models.py

```diff
@@ -1990,3 +1997,1360 @@ def export_data(
+            such model instance exists.
+        """
+        entity_id = cls.construct_id(
+            language_code, contributor_id)
+        return cls.get_by_id(entity_id)
+
+    @classmethod
+    def fetch_page(
+        cls,
+        page_size: int,
+        offset: int,
+        sort_by: Optional[str],
+        topic_ids: Optional[List[str]],
+        last_activity: Optional[int],
+        language_code: str
+    ) -> Tuple[Sequence[TranslationSubmitterTotalContributionStatsModel],
+                int,
+                bool]:
+        """Returns the models according to values specified.
+
+        Args:
+            page_size: int. Number of models to fetch.
+            offset: int. Number of results to skip from the beginning of all
+                results matching the query.
+            sort_by: str|None. A string indicating how to sort the result.
+            topic_ids: List[str]|None.
+                List of topics user has contributions.
+            last_activity: int. Number of days within which a user has
+                contributed.
+            language_code: str. Language Code for filter.
+
+        Returns:
+            3-tuple(sorted_results, next_offset, more). where:
+                sorted_results:
+                    list(TranslationSubmitterTotalContributionStatsModel).
+                    The list of models with supplied filters and sort_by.
+                next_offset: int. Number of results to skip in next batch.
+                more: bool. If True, there are (probably) more results after
+                    this batch. If False, there are no further results
+                    after this batch.
+        """
+        sort = -cls.recent_performance
+        if sort_by == (
+                constants.CD_ADMIN_STATS_SORT_OPTIONS[
+                    'IncreasingLastActivity']):
+            sort = -cls.last_contribution_date
+        elif sort_by == (
+                constants.CD_ADMIN_STATS_SORT_OPTIONS[
+                    'DecreasingLastActivity']):
+            sort = cls.last_contribution_date
+        elif sort_by == (
+                constants.CD_ADMIN_STATS_SORT_OPTIONS[
+                    'IncreasingPerformance']):
+            sort = cls.recent_performance
+        elif sort_by == (
+                constants.CD_ADMIN_STATS_SORT_OPTIONS[
+                    'DecreasingAccuracy']):
+            sort = -cls.overall_accuracy
+        elif sort_by == (
+                constants.CD_ADMIN_STATS_SORT_OPTIONS[
+                    'IncreasingAccuracy']):
+            sort = cls.overall_accuracy
+        elif sort_by == (
+                constants.CD_ADMIN_STATS_SORT_OPTIONS[
+                    'DecreasingSubmissions']):
+            sort = -cls.submitted_translations_count
+        elif sort_by == (
+                constants.CD_ADMIN_STATS_SORT_OPTIONS[
+                    'IncreasingSubmissions']):
+            sort = cls.submitted_translations_count
```

Comment:
else raise Exception?

Reply:
If None is passed, it will be set to -cls.recent_performance as default, as you can see on line 2203.